### PR TITLE
For initial release

### PR DESCRIPTION
**Pull request by Waseju**
First fully working version of the root tissue analysis pipeline.

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
+# nf-core/rts: Contributing Guidelines
```
Comment (ggabernet) [inline .github/CONTRIBUTING.md:1]: Removing all mentions of `nf-core` here to `qbic-pipelines` would be good

**Comment by ggabernet** [comment]
Hello, do we have any new developments here? So far the CI tests are not passing, also in the meantime there were multiple releases of nf-core tools so the template should probably be updated again

**Comment by luiskuhn** [comment]
This issue is fixed: https://github.com/qbic-pipelines/root-tissue-analysis/issues/6

File: modules/containers/ratioconv/Dockerfile [hunk 2]
```diff
@@ -6,10 +6,9 @@ RUN apt install -y xvfb
...
 RUN apt install libxrender1
 RUN apt install -y libxtst6
-RUN chmod +x entrypoint.sh
 RUN ImageJ-linux64 --headless --update add-update-site ImageScience http://sites.imagej.net/ImageScience/
 RUN ImageJ-linux64 --headless --update update jars/imagescience.jar
 RUN ImageJ-linux64 --headless --update update plugins/FeatureJ_.jar
 
 COPY ratio_macro.ijm ./
-ENTRYPOINT ["/entrypoint.sh"]
+ENTRYPOINT []
```
Comment (ggabernet) [inline modules/containers/ratioconv/Dockerfile]: would it maybe also work to completely remove this line?

Comment (ggabernet) [reply modules/containers/ratioconv/Dockerfile]:
```suggestion
```

**Review by ggabernet — APPROVED** [review]
Looks good to me now!

**Pull request merged**